### PR TITLE
Point website searches to /search/ instead of /apachesolr_search/

### PR DESCRIPTION
**Pull request by LucasCioffi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ### CHANGE LOG
 
 =======
+### v2.4.12
+> Point website searches to /search/ instead of /apachesolr_search/
```
Comment (ktp242) [inline CHANGELOG.md]: It is quite minor, but I think it is probably better use either "points" or "pointed" just to keep consistent with the verb tense?

**Comment by ktp242** [comment]
Just a minor suggestion. Other than that looks fine!

**Comment by LucasCioffi** [comment]
@EdwinGuzman I closed the last PR because it would merge into Master.  Is there a way to change that once it's created?

**Comment by LucasCioffi** [comment]
I'm closing this one because I saw in the README I need to name the branch with the `release-v.x.x.x` format.
Replaced by https://github.com/NYPL/dgx-header-component/pull/56

**Pull request closed without merging**